.github/workflows/publish.yaml

```
@@ -0,0 +1,78 @@
+name: '🏗️ Build and Publish jellyfin-ffmpeg'
```

```suggestion
name: '🚀 Build and Publish jellyfin-ffmpeg'
```

Additionally, any particular reason you explicitly exclude pre-releases?

---

Reason is that the prerelease would be pushed to the repo server - and we don't want a pre-release on out stable repos.

---

makes sense, so this is kind of a 'preparation' for if we ever actively intent to create per-releases?

---

[source](https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#release) I changed the trigger to `released` that should exclude all the cases we don't want.